Editor/Resources/EditorWindow/Pages/EC2Page.uxml

```diff
@@ -0,0 +1,92 @@
+﻿<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" noNamespaceSchemaLocation="../../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="./EC2Page.uss" />
+    <ui:VisualElement name="ManagedEC2Content" class="tab__content">
```

Comment:
`tab__content` should only be used by `GameLiftPlugin`. See how we did this with the landing page: https://github.com/aws/amazon-gamelift-plugin-unity/pull/94/files#diff-a7cce83e1068d1072c839330534ab2b18a78fdaff983022cc30402744931d080

Reply:
removed